config/storage/key-value/resource-store.json

```diff
@@ -56,6 +56,19 @@
           "PathBasedAuthorizer:_paths_value": { "@type": "DenyAllAuthorizer" }
         }
       ]
+    },
+    {
+      "comment": "Marks the /.internal/ storage container as an auxiliary resource, thereby hiding it from container representations.",
```

Comment:
I'm not sure that marking it as an auxiliary resource, just for the effect that has, is the right way.

`/.internal/` does otherwise not behave as an auxiliary resource, does it?

Reply:
Not really, since most other requests where auxiliary features are relevant will be blocked of by the authorizer preventing access to the `/.internal/` container.

I'm also not the biggest fan of this solution, so I'm even ok with reverting this, but it is currently the simplest way we have to hide resources. But it definitely is sort of hacky. I mostly went with it since it's only a config change and doesn't actually change any code. 

Another solution would be to have the `DataAccessorBasedStore` (or a new store type) take a list of regexes to ignore, but that class already does a lot.

Reply:
Fine with me—good to proceed.